tests/client/amo/test_utils.js

```diff
@@ -87,5 +92,35 @@ describe('amo/utils', () => {
           mockApi.verify();
         });
     });
+
+    it('returns a rejected Promise if the addonsType is wrong', () => {
+      ownProps.params.visibleAddonType = '404';
```

Comment:
I know this value can be anything but 404 is sort of confusing. I would suggest something like "addon-with-a-typo"